.gitmodules

```diff
@@ -4,12 +4,6 @@
 [submodule "vendor/github.com/coreos/go-systemd"]
 	path = vendor/github.com/coreos/go-systemd
 	url = https://github.com/coreos/go-systemd
-[submodule "vendor/github.com/Sirupsen/logrus"]
-	path = vendor/github.com/Sirupsen/logrus
-	url = https://github.com/Sirupsen/logrus
-[submodule "vendor/github.com/apex/log"]
-	path = vendor/github.com/apex/log
-	url = https://github.com/apex/log
```

Comment:
This was necessary because the packages that depend on `ecslogs/logrus` or `ecslogs/apex` would fail to compile otherwise.